### PR TITLE
DEV: Remove duplicated line of code

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: lib/post_destroyer.rb [hunk 1]
```diff
@@ -305,7 +305,6 @@ def mark_topic_changed
   def make_previous_post_the_last_one
     last_post =
       Post
-        .where("topic_id = ? and id <> ?", @post.topic_id, @post.id)
```
Comment (tgxworld) [inline lib/post_destroyer.rb:308]: O look this is the same line of code as two lines down :grin:

Comment (martin-brennan) [reply lib/post_destroyer.rb:308]: This is called "extra where" it's actually a valid SQL feature, look it up :P

**Review by martin-brennan — APPROVED** [review]

File: lib/post_destroyer.rb (hunk 1 above)
Comment (Drenmi) [reply lib/post_destroyer.rb:308]: There's also `limit(1).first`, but `#first` already adds a `limit(1)`. 🙂

**Comment by tgxworld** [comment]
Thank you for reviewing @martin-brennan

**Pull request merged**